exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/IndirectRowSet.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.drill.test.rowSet;
+package org.apache.drill.exec.physical.rowSet;
```

Comment:
Replace usage of `Sets.<Integer>newHashSet()`

Reply:
Thanks, replaced with `Collections.emptySet()`.